src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1164,6 +1167,7 @@ enum Provider {
     DOMINION("dominion", "Dominion"),
     ESS("ess", "ES&S"),
     HART("hart", "Hart"),
+    CSV("genericCsv", "CSV"),
```

Comment:
It occurred to me that we could do at least some barebones documentation on what this means in `hints_cvr_files.txt` under the `Provider` entry (e.g. break out a little * bulleted list of each one with a brief description). Can't really think of a better spot offhand.

Reply:
I'm not sure how to briefly describe these other than e.g. "Dominion: a Dominion CVR File". CSV could use better documentation, but I'm not sure what to write from the others...

Thoughts on just linking to example files for each? I think that may be more useful that trying to document it?